public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Mel's hack - the missing bits"
+author: "(David *)Frenkiel"
+date: Thu Jun 23 2022 01:14:31 GMT+0300
+---
+
+### A private _Mel moment_
```

Comment:
Titles should capitalize first letters (e.g. A Private Mel Moment).

Reply:
Titles markdown should be H2 (e.g. `##` instead of `###`)

Reply:
Please verify and resolve